Changelog.md

```diff
@@ -42,21 +42,12 @@
 
   If you encounter such cases, please adjust the type annotation.
 
-* The garbage collection scheduling algorithm is improved once again, and the
```

Comment:
if you change stuff here, better remove
`## 0.6.13 (2021-11-19)` from above, or change that to `0.6.14`.

Reply:
When releasing, I'll leave a note that `0.6.13` was pulled

Reply:
I'm offline now, everyone can push to my branch, could you do the tweaks please?

Reply:
Done